### ISSUE TITLE
how can I apply other loss functions

### ISSUE THREAD

**Issue by nepersica** [mateuszbuda/brain-segmentation-pytorch#30]
I'm thankful sharing pretrained weights.. I have a question for using.

I want to use other losses(ex.BCE, focal) on training.(not dice loss.)

When I use other functions, loss didn't decrease from particular value, but dice loss didn't.

Do the pretrained weights fit on dice loss only?
